src/components/CompareResults/SubtestsResults/SubtestsRevisionRow.tsx

```diff
@@ -28,7 +28,7 @@ const typography = style({
   fontFamily: 'SF Pro',
   fontStyle: 'normal',
   fontWeight: 400,
-  fontSize: '13px',
+  fontSize: '16px',
```

Comment:
We want to increase the font-size of the body of the entire app I believe. So please go to src/theme/components.js and update the `fontSize ` for `body1`.

Reply:
Might also have to add `body2: { fontSize: '16px',  }` in the `styleOverrides` for `Typography` in `components`.js.

Reply:
I updated the `fontSize` for `body1` indirectly fontsRaw.BodyDefault in `Fonts.ts` file, since that was in effect everywhere.